packages/frontend/src/components/DataViz/hooks/useChartViz.ts

```diff
@@ -7,14 +7,15 @@ import {
     isVizTableConfig,
     PieChartDataModel,
     type AllVizChartConfig,
+    type IResultsRunner,
     type PivotChartData,
+    type VizChartLayout,
 } from '@lightdash/common';
 import { useQuery } from '@tanstack/react-query';
 import { useMemo } from 'react';
 import { useOrganization } from '../../../hooks/organization/useOrganization';
-import { type ResultsRunner } from '../transformers/ResultsRunner';
 
-type Args<T extends ResultsRunner> = {
+type Args<T extends IResultsRunner<VizChartLayout>> = {
```

Comment:
`VizChartLayout` is runner specific so there shouldn't be an `type VizChartLayout` imported here. Are we able to have `T, V extends IResultsRunner<V>` or similar? It should be opaque here